lib/primer/view_components/linters/helpers.rb

```diff
@@ -13,10 +13,13 @@ module Helpers
         link menuitem meta param source track wbr img
       ].freeze
 
+      DUMP_FILE = ".erblint-counter-ignore.json"
```

Comment:
Could we put this in the /static dir?

Reply:
We could, but the goal here isn't to keep the file. The idea is that this file will be generated when reporting data to Datadog and then removed. This allow us to enable specific linters when reporting metrics without having them always enables (and causing noise with ignore comments in all files)
We also don't want to commit this file since it will generate **tons** of merge conflicts